### PR TITLE
OP-464 use isEmpty()

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-464

### CODE REVIEW

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ public void actionPerformed(ActionEvent e) {
 	private boolean isValidPaymentDate(GregorianCalendar datePay) {
 		GregorianCalendar now = new GregorianCalendar();
 		GregorianCalendar lastPay = new GregorianCalendar();
-	    if (payItems.size() > 0) { 
+	    if (!payItems.isEmpty()) {
 	    	lastPay = payItems.get(payItems.size()-1).getDate();
 		} else {
 			lastPay = billDate;
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/gui/PatientBillEdit.java:1379]: Strange indentation

Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/gui/PatientBillEdit.java:1379]: I'll fix it; it is most likely a tabs vs. spaces issue.

Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/gui/PatientBillEdit.java]: Reformatted the entire method so it should be consistent.

Comment (mwithi) [reply src/main/java/org/isf/accounting/gui/PatientBillEdit.java]: Thanks!

**Pull request merged**